### ISSUE TITLE
'Problem with shortcut' popup on world generation

### ISSUE THREAD

**Issue by Pulz-TCP** [Hunternif/AntiqueAtlas#226]
Fairly sure this is related to #213 

If I have _any_ broken links on my desktop, when I create a new world it will hang just before I join the world, with a popup telling me the link(s) are broken. Only after pressing cancel will I load into the world.

I have been able to reproduce this 100% of the time, by creating a shortcut to an empty .jar file and then deleting the linked .jar file.

https://i.imgur.com/QeLl1tZ.gif

This is definitely AntiqueAtlas, the issue is not present without it.

Forge: 2838
AtiqueAtlas: 4.5.1
Mod list: https://gist.github.com/PulzHF/0184d147ed6140442674106b854ec8bd
(Files named -jar.jar due to my Solder modpack system, all legitimate from their respective sources)

**Comment by Pulz-TCP** [comment]
Any news on this one? Would really like to add it to my modpack :)

**Comment by Hunternif** [comment]
Hi PulzHF.

This is really weird, but maybe it's somehow related to this issue: https://bugs.java.com/bugdatabase/view_bug.do?bug_id=5049016. Though it should have been fixed 15 years ago.

We export images to the screenshots folder now, so there's no need to use JFileChooser. The reference was still in the code, but now I deleted it. Try this build (rename it to .jar): [antiqueatlas-1.12.2-4.5.2.zip](https://github.com/AntiqueAtlasTeam/AntiqueAtlas/files/3694689/antiqueatlas-1.12.2-4.5.2.zip)


Also, can I ask you to try it on other machines?
What is your Java version? To get the version, open Command Prompt and type `java -version`
